### ISSUE TITLE
Installation with pip requires --process-dependency-links

### ISSUE THREAD

**Issue by andrewcchen** [polysquare/polysquare-travis-container#16]
```
$ pip install git+http://github.com/polysquare/polysquare-travis-container
Collecting git+http://github.com/polysquare/polysquare-travis-container
  Cloning http://github.com/polysquare/polysquare-travis-container to /tmp/pip-uhfcbqy0-build
Requirement already satisfied (use --upgrade to upgrade): setuptools in /usr/lib/python3.4/site-packages (from psqtraviscontainer==0.0.2)
Collecting configargparse (from psqtraviscontainer==0.0.2)
  Using cached ConfigArgParse-0.9.3.tar.gz
Collecting pycurl (from psqtraviscontainer==0.0.2)
  Using cached pycurl-7.19.5.1.tar.gz
Collecting python-debian (from psqtraviscontainer==0.0.2)
  Using cached python-debian-0.1.23.tar.gz
Requirement already satisfied (use --upgrade to upgrade): six in /usr/lib/python3.4/site-packages (from psqtraviscontainer==0.0.2)
Collecting urlgrabber==3.10.1 (from psqtraviscontainer==0.0.2)
  Could not find a version that satisfies the requirement urlgrabber==3.10.1 (from psqtraviscontainer==0.0.2) (from versions: 3.1.0, 3.9.1)
  No matching distribution found for urlgrabber==3.10.1 (from psqtraviscontainer==0.0.2)
```

**Comment by smspillaz** [comment]
Hey @llkiwi2006 ,

At the moment I'm using dependency links to install urlgrabber, since the version on PyPI is way out of date and doesn't implement some of the features that I'm using (like python 3 support). You'll need to specify `--follow-dependency-links`.

I'm not too sure what to do about the outdated dependency.

**Comment by smspillaz** [comment]
@llkiwi2006 I've updated the documentation to specify this. Let me know if this works for you and I'll keep the issue open as a reference for everyone else.

**Comment by llkiwi2006** [comment]
Thanks, it installs now.

**Comment by smspillaz** [comment]
I'll keep this open for reference :)

**Comment by smspillaz** [comment]
Closed #16 via #30
